src/test/java/com/exasol/dbbuilder/dialects/AbstractDatabaseObjectCreationAndDeletionIT.java

```diff
@@ -23,7 +23,7 @@
     protected Connection adminConnection;
 
     @BeforeEach
-    void beforeEach() throws SQLException {
+    protected void beforeEach() throws SQLException {
```

Comment:
This should not be necessary, better use default visibility if possible

Reply:
fixed